middleware/validator/params/requestValidator.js

```diff
@@ -0,0 +1,40 @@
+import { RequestStatus } from "../../../model/enun/requestStatus.js";
+
+function validatorSatus(req, res, next){
+  const status = req.query.status;
+  const validStatus = Object.values(RequestStatus);
+ 
+  if (!status) {
+    next()
+  } else if (validStatus.includes(status)) {
+      next()
+    } else {
+      res.status(400).send({ error: "Invalid Status param value" });
```

Comment:
```suggestion
      res.status(400).send({ error: `Invalid Status param value: ${status}` });
```
agregar el estado incorrecto que se mandó

Reply:
ok